src/Sarif.Converters.UnitTests/SemmleQLConverterTests.cs

```diff
@@ -86,7 +86,7 @@ public void SemmleQLConvert_EmbeddedLocations()
       ""results"": [
         {
           ""message"": {
-            ""text"": ""Call to Equals() comparing incomparable types[IComparable] and [ClientAttributeValue]""
+            ""text"": ""Call to Equals() comparing incomparable types[IComparable](1) and [ClientAttributeValue](1)""
```

Comment:
The second link should be `(2)`. But... does it make sense to link into a DLL? The user will click on it and then what?

Reply:
Yeah I don't know how to handle this

Reply:
Maybe we need to wait until they add (complete) support for this to the Semmle log format.

Reply:
Well, if that hyperlink is part of the original message in the Semmle output file, then there's nothing you can do. Let me know if it is and I'll mention it to Luke and Michael.

What about that wrong index on the second link?

Reply:
They provide a list of related links. See line 177 in the converter for an example.

Looking at the index now.

Reply:
That example looks wrong:
```
            // The rawMessage contains embedded related locations. We need to extract the related locations and reformat the rawMessage embedded links wrapped in [brackets].
            // Example rawMessage
            //     po (coming from [["hbm"|"relative://windows/Core/ntgdi/gre/brushapi.cxx:176:4882:3"],["hbm"|"relative://windows/Core/ntgdi/gre/windows/ntgdi.c:1873:50899:3"],["hbm"|"relative://windows/Core/ntgdi/gre/windows/ntgdi.c:5783:154466:3"]]) may not have been checked for validity before call to vSync.
            // Example normalizedMessage
            //     po (coming from [hbm]) may not have been checked for validity before call to vSync.
```
1. It doesn't show the link numbers.
2. The original message has three related locations, but the resulting message only shows one.

Reply:
1. Right, this is what I mean by complete support. I believe Luke said that was coming soon, don't know which version though.

Reply:
2. Yes that's why I was linking to the first location in the list. You are right, I need to change it to use location 2 for the second link.

Reply:
Fixed